src/Modules/MapsModule/Controllers/MapsController.cs

```diff
@@ -71,14 +77,23 @@ public async Task RemoveMap(long mapId)
             return;
         }
 
-        await _mapService.RemoveMapAsync(mapId);
+        try
+        {
+            await _mapService.RemoveMapAsync(mapId);
+        }
+        catch (Exception)
+        {
+            await _server.ErrorMessageAsync(_locale.PlayerLanguage.MapRemovedFailed(mapId), Context.Player);
+            throw;
+        }
 
         Context.AuditEvent.Success()
             .WithEventName(AuditEvents.MapRemoved)
-            .HavingProperties(new {Map = map})
+            .HavingProperties(new { Map = map })
             .Comment(_locale.Audit_MapRemoved);
-        
-        await _server.SuccessMessageAsync(_locale.PlayerLanguage.MapRemovedSuccessfully(map.Name, map.Author.NickName), Context.Player);
+
+        await _server.SuccessMessageAsync(_locale.PlayerLanguage.MapRemovedSuccessfully(map.Name, map.Author.NickName),
+            Context.Player);
         _logger.LogInformation("Player {PlayerId} removed map {MapName}", Context.Player.Id, map.Name);
```

Comment:
```suggestion
        _logger.LogDebug("Player {PlayerId} removed map {MapName}", Context.Player.Id, map.Name);
```
Redundant for "info" level logs. should be used for debugging instead. The audit log already outputs this in the log and database anyways.